app-tracking-protection/vpn-impl/src/main/res/values/strings-vpn.xml

```diff
@@ -196,12 +186,12 @@
     <string name="atp_ActivityPastWeek">Past 7 Days</string>
     <string name="atp_ActivityTitle">App Tracking Protection</string>
     <string name="atp_MostRecentActivityTitle">All Recent Activity</string>
-    <string name="atp_ActivityRecentActivity">Recent Activity</string>
+    <string name="atp_ActivityRecentActivity">Activity</string>
     <string name="atp_ActivityCtaShowAll">View All Recent Activity</string>
     <string name="atp_ActivityAppTrackersFAQ">What Are App Trackers?</string>
     <string name="atp_ActivityReportProblem">Report a problem With an App</string>
     <string name="atp_ActivityBetaInstructions">Beta Instructions</string>
-    <string name="atp_ActivityManageProtection">Having Problems With An App?</string>
+    <string name="atp_ActivityManageProtection">Having Issues with an app?</string>
```

Comment:
We've modified the content of this string but left the ID intact. But we haven't removed the translated strings.
The string is also in string-vpn-untranslated.xml.
Translations are not going to be correct in the end